src/components/preferences/styles.js

```diff
@@ -0,0 +1,70 @@
+export default (theme) => ({
+    root: {
+        backgroundColor: theme.palette.background.paper,
+        width: "100%",
+    },
+
+    preferences: {
+        fontSize: 14,
+    },
+
+    toggle: {
+        float: "right",
+        marginRight: "10%",
+    },
+
+    sectionHeader: {
+        fontFamily: "roboto",
+        marginLeft: "2%",
+        fontSize: 20,
+    },
+
+    textField: {
+        width: "60%",
+        marginLeft: "5%",
+    },
+
+    webhooksTextField: {
+        width: "50%",
+    },
+
+    dropDown: {
+        width: "20%",
+        marginLeft: "3%",
+        marginRight: "3%",
+    },
+
+    actionButton: {
+        fontSize: 16,
+        marginTop: 5,
+        marginLeft: 5,
```

Comment:
Just picking a random spot, instead of setting these margins to specific numbers, I would use the [theme's spacing](https://material-ui.com/customization/spacing/) instead.  So lots of `marginTop: theme.spacing(1)`, for example.